src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -865,22 +870,13 @@ public void parse(InputFile inputFile, Format ext, int type, boolean thumbOnly,
 						ffmpeg_parsing = true;
 					}
 				}
-				
-				if (configuration.getImageThumbnailsEnabled() && thumbOnly) {
+
+				if (configuration.isThumbnailGenerationEnabled() && configuration.getImageThumbnailsEnabled() && thumbOnly) {
 					LOGGER.trace("Creating (temporary) thumbnail: {}", file.getName());
 
 					// Create the thumbnail image using the Thumbnailator library
 					try {
-						int thumbnailWidth = renderer.isSquareImageThumbnails() ? renderer.getThumbnailHeight() : renderer.getThumbnailWidth();
-						int thumbnailHeight = renderer.getThumbnailHeight();
-
-						// Make sure the image fits in the renderer's bounds
-						boolean isFullyPlayedThumbnail = FullyPlayed.isFullyPlayedThumbnail(file);
-						thumb = UMSUtils.scaleImage(Files.readAllBytes(file.toPath()), thumbnailWidth, thumbnailHeight, isFullyPlayedThumbnail, renderer);
-
-						if (isFullyPlayedThumbnail) {
-							thumb = FullyPlayed.addFullyPlayedOverlay(thumb, MediaType.IMAGE);
-						}
+						thumb = ImagesUtil.scaleImage(Files.readAllBytes(file.toPath()), 320, 320, ScaleType.MAX, ImageFormat.JPEG, false);
```

Comment:
@valib Looking at ```ffmpeg_parsing``` I can't see much logic at all. What is this supposed to do? It's set to false for all audio, yet later on (in audio) there's conditions for whether it is true or false.